packages/strapi-admin/admin/src/containers/App/utils/unique-identifier.js

```diff
@@ -0,0 +1,10 @@
+import FingerprintJS from '@fingerprintjs/fingerprintjs';
+
+const getUniqueIdentifier = async () => {
```

Comment:
I think that we could add this unique identifier in the package.json of a generated app, you'll be able to retrieve it in the admin since we are requiring the package.json.

WDYT?

Reply:
We have to generate the identifier from the frontend, we cannot generate it from the backend as it is based on the browser's public exposed parameters. We only need it once for now so I don't think it is necessary to store it somewhere. Also, the ID might change over time (e.g. the user uses a new browser)